Makefile

```diff
@@ -78,7 +78,7 @@ gen-border-radius-styles:
 	@echo "Generating border-radius style definitions..."
 	@node genBorderRadius.js
 
-LINT_MAX_LESS_PROBLEMS := 102
+LINT_MAX_LESS_PROBLEMS := 96
```

Comment:
did this introduce some more lint issues? 😅

Reply:
It introduced a false positive actually! And it didn't feel right to appease the linter with a fix for an incorrectly identified issue, so I instead updated stylelint to mitigate the false positive.

Here was the error before I upgraded stylelint:

```
src/AnnouncementBubble/NormalAnnouncementBubble.less
 181:3  ✖  Expected selector ".NormalAnnouncementBubble--translatedLanguages--container .margin--top--m()" to come before selector ".Button.NormalAnnouncementBubble--replyButton .margin--top--m()"   no-descending-specificity
```

It's subtle, but stylelint was misinterpreting our `.margin--top--m` mixin as a CSS selector. So it was stating that `.NormalAnnouncementBubble--translatedLanguages--container .margin--top--m` was of lower specificity that `.Button.NormalAnnouncementBubble--replyButton .margin--top--m` but in truth, the two selectors are unrelated.

`.NormalAnnouncementBubble--translatedLanguages--container ` is independent of `.Button.NormalAnnouncementBubble--replyButton` -- they share no overlap.

Reply:
Ahh, nice investigation! Good call